code/modules/paperwork/ticketmachine.dm

```diff
@@ -102,14 +103,18 @@
 	handle_maptext()
 
 /obj/machinery/ticket_machine/proc/handle_maptext()
-	switch(ticket_number) //This is here to handle maptext offsets so that the numbers align.
-		if(0 to 9)
-			maptext_x = 13
-		if(10 to 99)
-			maptext_x = 10
-		if(100)
-			maptext_x = 8
-	maptext = "<font face='Small Fonts'>[ticket_number]</font>"
+	if(disabled)
+		maptext_x = 13
+		maptext = "X"
```

Comment:
Huh? Is the X even ever visible? Can you share a screenshot? I don't see it in the gif in the PR description.

Reply:
yes I took gif before I added X to display. 
![image](https://user-images.githubusercontent.com/16618648/98432934-6f4eac00-2088-11eb-85d8-5d3e18548bf5.png)

Reply:
I think I might acutally make it font face='small fonts' or whatever, so it doesnt look garbo.

Reply:
FWIW, I think blank display also looks fine (but i'm not a graphic design person)

Reply:
![dreamseeker_sWzMJ0WrZE](https://user-images.githubusercontent.com/16618648/98433161-197b0380-208a-11eb-8947-d5ffba4807ce.png)
Beauteous!